applications/sdp/mspi/src/main.c

```diff
@@ -306,6 +307,13 @@ void prepare_and_read_data(nrfe_mspi_xfer_packet_msg_t *xfer_packet, volatile ui
 	xfer_params.xfer_data[HRT_FE_ADDRESS].data = (uint8_t *)&xfer_packet->address;
 	xfer_params.xfer_data[HRT_FE_ADDRESS].word_count = nrfe_mspi_xfer_config_ptr->address_length;
 
+	/* Configure dummy_cycles phase. */
+	xfer_params.xfer_data[HRT_FE_DUMMY_CYCLES].vio_out_set =
+		nrf_vpr_csr_vio_out_buffered_reversed_word_set;
+	xfer_params.xfer_data[HRT_FE_DUMMY_CYCLES].data = (uint8_t *)&dummy_cycles_data;
+	xfer_params.xfer_data[HRT_FE_DUMMY_CYCLES].word_count =
+		nrfe_mspi_xfer_config.rx_dummy * xfer_params.bus_widths.dummy_cycles;
```

Comment:
There should most probably be an assert in dev_config function checking if rx_dummy is dividable by 8. For now, other values of rx_dummy will not work, because it would require changing SHIFTCNTB value.

Reply:
Added